### ISSUE TITLE
Question about loop Configuration for ACT Algorithm in RLBench Dataset

### ISSUE THREAD

**Issue by Fisher-Wang** [HaoyiZhu/PointCloudMatters#1]
Thank you for your amzing work providing a comprehensive benchmark and well-structured codebase, which is really inspiring!

I have a question regarding the configuration of the ACT Algorithm on the RLBench dataset. I noticed that the `loop` variable for the training dataset is set to 100, which lead to the data in one epoch repeating 100 times.

https://github.com/HaoyiZhu/PointCloudMatters/blob/7feae7b0f82634d0cff795fdb2118c3ce6d8c35f/configs/exp_rlbench_act_policy/base.yaml#L31

Could you explain the reasoning behind this approach instead of simply multiplying the epoch number by 100 (and maybe change the scheduler parameters accordingly)? I've not encountered dataset configuration done this way before.

Thank you for your help!

**Comment by HaoyiZhu** [comment]
Hi, thanks for pointing out. That's indeed a type and should be set to 1. I have updated the code and you can pull the latest version.

By default, now we use `loop=1`, `max_epochs=2000`, `check_val_every_n_epoch=100`, which is equal to `loop=100`, `max_epochs=20`, `check_val_every_n_epoch=1`. You do not need to modify scheduler parameters since we use per-step scheduler.

As for why we have a `loop` option, we have noticed that it may take some time for dataloader initialization at the beginning of each epoch. So when the epoch number is large, the total training time may be longer. A `loop` parameter can force the dataloader to be re-initialized less frequently and fasten the overall training speed.

**Comment by Fisher-Wang** [comment]
Thank you for the detailed explanation! I appreciate the clarification.
